src/view/components/objectEditor/helpers/object-analytics/populateTreeNode.js

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import computePopulationNote from "../computePopulationNote";
```

Comment:
The `analytics` circle defaults to blue even though the form is empty:

![Screenshot 2024-03-08 at 1 42 30 PM](https://github.com/adobe/reactor-extension-alloy/assets/1738196/3240c81f-d899-4a60-b2c2-99b963bd4e86)